Ch8.1 #2,9 Ch8.2 #1,9,10 Ch8.3 #6,9

# Chapter 8.1

**2. Suppose that a random sample is to be taken from the normal distribution with unknown mean θ and standard deviation 2. How large a random sample must be taken in order that $E_θ(|\overline{X}_n − θ|^2) ≤ 0.1$ for every possible value of θ?**

The sampling distribution of the sample mean is given by,

$$
\overline{X}_n \sim \text{Normal}(\theta, {\sigma^2 \over n})
$$

Here, the standard deviation is known. So,

$$
\overline{X}_n \sim \text{Normal}(\theta, {4 \over n})
$$

Let,

$$
\epsilon = \overline{X}_n - \theta
$$

Then,

$$
\epsilon \sim \text{Normal}(0, {4 \over n})
$$

By the definition of variance,

$$
\operatorname{var}(\epsilon) = E[\epsilon^2] - E[\epsilon]^2
$$

The mean and variance of $\epsilon$ are known,

$$
{4 \over n} = E[\epsilon^2]
$$

Suppose,

$$
E[(\overline{X}_n - \theta)^2] \le 0.1
$$

Then,

$$
{4 \over n} \le 0.1
$$

$$
{4 \over 0.1} \le n
$$

$$
n \ge 40
$$

**9. Let $X_1, . . . , X_n$ be a random sample from the exponential distribution with parameter $θ$. Find the c.d.f. for the sampling distribution of the M.L.E. of $θ$. (The M.L.E. itself was found in Exercise 7 in Sec. 7.5.)**

We have that,

$$
X_1, \dots, X_n \sim \text{Exponential}(\theta)
$$

The MLE of $\theta$, $\hat{\theta}$ is given by,

$$
\hat{\theta} = {n \over \sum_{i = 1}^n X_i} = \overline{X}_n^{-1}
$$

# Chapter 8.2

**1. Suppose that we will sample 20 chunks of cheese in Example 8.2.3. Let $T = \sum_{i=1}^{20} (X_i− μ)^2/20$, where $X_i$ is the concentration of lactic acid in the ith chunk. Assume that $σ^2 = 0.09$. What number $c$ satisfies $Pr(T ≤ c)= 0.9$?**

We have that,

$$
T = \sum_{i = 1}^{20} (X_i - \mu)^2 /20
$$

$$
T = {\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2}
$$

But,

$$
{X_i - \mu \over \sigma} \sim \text{Normal}(0, 1)
$$

So,

$$
{(X_i - \mu)^2 \over \sigma^2} \sim \chi^2(1)
$$

So,

$$
\sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \chi^2(20) = \text{Gamma}(10, 1/2)
$$

Since we know $\sigma^2$, we can find,

$$
{\sigma^2 \over 20} = {0.09 \over 20} = 0.0045
$$

So,

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim 0.0045 \text{Gamma}(10, 1/2)
$$

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \text{Gamma}(10, 1/((2)0.0045))
$$

$$
{\sigma^2 \over 20} \sum_{i = 1}^{20} {(X_i - \mu)^2 \over \sigma^2} \sim \text{Gamma}(10, 0.009)
$$

$$
T \sim \text{Gamma}(10, 0.009)
$$

We need the inverse CDF of $\text{Gamma}(10, 0.009)$ applied to 0.9.

In [10]:
from scipy import stats

stats.gamma(a = 10, scale = 1/0.009).ppf(0.9)

np.float64(1578.4433657947573)

So,

$$
c = 1578.4433657947573
$$

So that,

$$
P(T \le C) = 0.9
$$

**9. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with mean $\mu$ and variance $σ^2$. Find the distribution of,**

$$
{n(\overline{X}_n - \mu)^2 \over \sigma^2}
$$

The sample mean of a sample from the normal distribution has mean $\mu$ and variance $\sigma^2/n$. (I'll be parametrizing the normal with mean and variance in this problem.)

$$
\overline{X}_n \sim \text{Normal}(\mu, \sigma^2/n)
$$

So,

$$
\overline{X}_n - \mu \sim \text{Normal}(0, \sigma^2/n)
$$

So,

$$
{\overline{X}_n - \mu \over \sigma} \sim \text{Normal}(0, {1 \over \sigma^2}{\sigma^2 \over n})
$$

$$
{\overline{X}_n - \mu \over \sigma^2} \sim \text{Normal}(0, {1 \over n})
$$

So,

$$
\sqrt{n}{\overline{X}_n - \mu \over \sigma^2} \sim \text{Normal}(0, n{1 \over n})
$$

$$
\sqrt{n}{\overline{X}_n - \mu \over \sigma^2} \sim \text{Normal}(0, 1)
$$

The square of a standard normal variable is a $\chi^2(1)$. So,

$$
\left(\sqrt{n}{\overline{X}_n - \mu \over \sigma^2}\right)^2 \sim \chi^2(1)
$$

$$
n{(\overline{X}_n - \mu)^2 \over \sigma^2} \sim \chi^2(1)
$$

**10. Suppose that six random variables $X_1, . . . , X_6$ form a random sample from the standard normal distribution, and let,**

$$
Y = (X_1 + X_2 + X_3)^2 + (X_4 + X_5 + X_6)^2
$$

**Determine a value of $c$ such that the random variable $cY$ will have a $χ^2$ distribution.**

The $X_i$ are independent samples. So, when we add them together we get a normal that sums the means and variances. (I'll be parametrizing the Normal with mean and variance in this problem.)

$$
X_1 + X_2 + X_3 \sim \text{Normal}(0, 3)
$$

So,

$$
{1 \over \sqrt{3}}(X_1 + X_2 + X_3) \sim \text{Normal}(0, \left({1 \over \sqrt{3}}\right)^23)
$$

So,

$$
{1 \over \sqrt{3}}(X_1 + X_2 + X_3) \sim \text{Normal}(0, 1)
$$

The square of the standard normal is the $\chi^2(1)$. So,e

$$
\left({1 \over \sqrt{3}}(X_1 + X_2 + X_3)\right)^2 \sim \chi^2(1)
$$

$$
{1 \over 3}(X_1 + X_2 + X_3)^2 \sim \chi^2(1)
$$

Similarly,

$$
{1 \over 3}(X_4 + X_5 + X_6)^2 \sim \chi^2(1)
$$

The sum of two $\chi^2$ variables is distributed according to a $\chi^2$ with the sums of their degrees of freedom.

$$
{1 \over 3}(X_1 + X_2 + X_3)^2 + {1 \over 3}(X_4 + X_5 + X_6)^2 \sim \chi^2(2)
$$

$$
{1 \over 3}\left( (X_1 + X_2 + X_3)^2 + (X_4 + X_5 + X_6)^2 \right) \sim \chi^2(2)
$$

$$
{1 \over 3}Y \sim \chi^2(2)
$$

So, when $c = 1/3$, $cY$ is $\chi^2$ distributed.

# Chapter 8.3

**6. Suppose that $X_1, . . . , X_n$ form a random sample from the normal distribution with mean $μ$ and variance $σ^2$. Assuming that the sample size $n$ is 16, determine the values of the following probabilities:**

**(a) $P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \mu)^2 \le 2\sigma^2)$**

We have that,

$$
X_i \sim \text{Normal}(\mu, \sigma^2)
$$

Then,

$$
X_i - \mu \sim \text{Normal}(0, \sigma^2)
$$

$$
{X_i - \mu \over \sigma} \sim \text{Normal}(0, 1)
$$

$$
\left({X_i - \mu \over \sigma}\right)^2 \sim \chi^2(1)
$$

The sum of independent $\chi^2$ variables is a $\chi^2$ with the sum of the degrees of freedom.

$$
\sum_{i = 1}^n \left({X_i - \mu \over \sigma}\right)^2 \sim \chi^2(n)
$$

$$
{1 \over \sigma^2} \sum_{i = 1}^n \left(X_i - \mu\right)^2 \sim \chi^2(n)
$$

Assume $n \ne 0$,

$$
{n \over \sigma^2}{1 \over n} \sum_{i = 1}^n \left(X_i - \mu\right)^2 \sim \chi^2(n)
$$

Let,

$$
\hat{\sigma}^2 = {1 \over n} \sum_{i = 1}^n \left(X_i - \mu\right)^2
$$

Then,

$$
{n \over \sigma^2}\hat{\sigma}^2 \sim \chi^2(n)
$$

Since ${n \over \sigma^2}$, multiplication by this quantity constitutes a monotone increasing transform. So,

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
P({n \over \sigma^2}{\sigma^2 \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le {n \over \sigma^2}2\sigma^2)
$$

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
P({n \over 2} \le {n \over \sigma^2}\hat{\sigma}^2 \le 2n)
$$

Let $\Phi_n$ denote the CDF of the $\chi^2(n)$. Then,

$$
P({\sigma^2 \over 2} \le \hat{\sigma}^2 \le 2\sigma^2) =
\Phi_n(2n) - \Phi_n(n/2)
$$

So,

$$
P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \mu)^2 \le 2\sigma^2) =
\Phi_n(2n) - \Phi_n(n/2)
$$

The $\chi^2$ CDF has no closed form, but we can evaluate this quantity numerically when $n = 16$.

In [12]:
n = 16
stats.chi2(df=n).cdf(2*n) - stats.chi2(df=n).cdf(n/2)

np.float64(0.9388666032540479)

So,

$$
P(\sigma^2{1 \over 2} \le {1 \over 16}\sum_{i = 1}^{16} (X_i - \mu)^2 \le 2\sigma^2) =
0.9388666032540479
$$

**(b) $P(\sigma^2{1 \over 2} \le {1 \over n}\sum_{i = 1}^n (X_i - \overline{X}_n)^2 \le 2\sigma^2)$**

$$
\sum_{i = 1}^n (X_i - \overline{X}_n)^2 = \sum_{i = 1}^n \left( X_i^2 - 2X_i\overline{X}_n + \overline{X}_n^2 \right)
$$

$$
\sum_{i = 1}^n (X_i - \overline{X}_n)^2 = \sum_{i = 1}^n X_i^2 - 2\sum_{i = 1}^n X_i\overline{X}_n + n  \overline{X}_n^2
$$

$$
\sum_{i = 1}^n (X_i - \overline{X}_n)^2 = n \overline{X}_n^2 + \sum_{i = 1}^n X_i^2 - 2\sum_{i = 1}^n X_i {1 \over n} \sum_{j = 1}^n X_j
$$

Take these terms one at a time.

$$
X_i^2 \sim \chi^2(1)
$$

$$

$$

We have that,

$$
\overline{X}_n = {1\over n}\sum_{j = 1}^n X_j
$$

So,

$$
X_i - \overline{X}_n = X_i - {1\over n}\sum_{j = 1}^n X_j
$$

$$
X_i - \overline{X}_n = {n - 1 \over n}X_i - \sum_{j \in \{1...n\}/i} {1\over n}X_j
$$

This is a linear combination of independent gaussians, so its gaussian. Take the expectation.

$$
E[X_i - \overline{X}_n] = E[{n - 1 \over n}X_i] - {1\over n}\sum_{j \in \{1...n\}/i} E[X_j]
$$

$$
E[X_i - \overline{X}_n] = {n - 1 \over n}\mu - {1\over n}\sum_{j \in \{1...n\}/i}\mu
$$

$$
E[X_i - \overline{X}_n] = {n - 1 \over n}\mu - {n - 1\over n}\mu
$$

$$
E[X_i - \overline{X}_n] = 0
$$

Take the variance.

$$
\operatorname{var}[X_i - \overline{X}_n] = \operatorname{var}\left[{n - 1 \over n}X_i - {1\over n}\sum_{j \in \{1...n\}/i} X_j\right]
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {(n - 1)^2 \over n^2}\operatorname{var}[X_i] + {1\over n^2}\sum_{j \in \{1...n\}/i} \operatorname{var}\left[X_j\right]
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {(n - 1)^2 \over n^2}\sigma^2 + {1\over n^2}\sum_{j \in \{1...n\}/i} \sigma^2
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {(n - 1)^2 \over n^2}\sigma^2 + {n - 1\over n^2} \sigma^2
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {(n - 1)^2 + (n - 1) \over n^2}\sigma^2
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {n(n - 1) \over n^2}\sigma^2
$$

$$
\operatorname{var}[X_i - \overline{X}_n] = {n - 1 \over n}\sigma^2
$$

So,

$$
X_i - \overline{X}_n \sim \text{Normal}(0, {n - 1 \over n}\sigma^2)
$$

**9. Suppose that each of two statisticians, $A$ and $B$, independently takes a random sample of 20 observations from the normal distribution with unknown mean $μ$ and known variance 4. Suppose also that statistician $A$ finds the sample variance in his random sample to be 3.8, and statistician $B$ finds the sample variance in her random sample to be 9.4. For which random sample is the sample mean likely to be closer to the unknown value of $μ$?**